x-pack/test/functional/page_objects/search_navigation.ts

```diff
@@ -5,30 +5,21 @@
  * 2.0.
  */
 
-import { FtrProviderContext } from '../ftr_provider_context';
+import type { FtrProviderContext } from '../ftr_provider_context';
 
 export function SearchNavigationProvider({ getService, getPageObjects }: FtrProviderContext) {
   const retry = getService('retry');
-  const PageObjects = getPageObjects(['common']);
+  const { common, indexManagement, header } = getPageObjects([
+    'common',
+    'indexManagement',
+    'header',
+  ]);
   const testSubjects = getService('testSubjects');
 
   return {
-    async navigateToLandingPage() {
-      await retry.tryForTime(60 * 1000, async () => {
-        await PageObjects.common.navigateToApp('landingPage');
-        // Wait for the side nav, since the landing page will sometimes redirect to index management now
-        await testSubjects.existOrFail('svlSearchSideNav', { timeout: 2000 });
-      });
-    },
-    async navigateToGettingStartedPage() {
-      await retry.tryForTime(60 * 1000, async () => {
-        await PageObjects.common.navigateToApp('serverlessElasticsearch');
-        await testSubjects.existOrFail('svlSearchOverviewPage', { timeout: 2000 });
-      });
-    },
     async navigateToElasticsearchStartPage(expectRedirect: boolean = false) {
       await retry.tryForTime(60 * 1000, async () => {
-        await PageObjects.common.navigateToApp('elasticsearchStart', {
```

Comment:
FYI this will work you just have to add entries for the app id to the config, [see here](https://github.com/elastic/kibana/blob/main/x-pack/test/functional/config.base.js#L192-L194), that could be done in the base config or the functional search config specifically like we do for serverless, [see here](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/test_suites/search/config.ts#L33).

by doing `await common.navigateToApp('elasticsearch/start'` you're using the fallback behavior to navigate to `/app/{appId}` passed since it can't find `elasticsearch/start` in the `apps: {}` map from the config.

Reply:
Yeah, I wasn't sure what is better to do. I'll it to `config.base` then